master/reference/calicoctl/resources/policy.md

```diff
@@ -92,8 +92,10 @@ applied action is deny.
 |-------------|-----------------------------|-------------------|--------|------------|
 | tag (deprecated)      | Match on tag. |  | string | |
 | notTag (deprecated)   | Negative match on tag. |  | string | |
-| net    | Match on CIDR. | Valid IPv4 or IPv6 CIDR  | cidr | |
-| notNet | Negative match on CIDR. | Valid IPv4 or IPv6 CIDR | cidr | |
+| net                   | Deprecated (use "nets" instead): Match on CIDR. | Valid IPv4 or IPv6 CIDR  | cidr | |
+| nets                  | Match any one of the listed CIDRs. | List of valid IPv4 or IPv6 CIDRs  | list of cidr |
+| notNet                | Deprecated (use "notNets" instead): Negative match on CIDR. | Valid IPv4 or IPv6 CIDR | cidr | |
+| nets                  | Negative match on CIDRs. | List of valid IPv4 or IPv6 CIDRs  | list of cidr |
```

Comment:
Ah, some subtlety here: I meant "invalid" as in "these are the CIDRs that I want to disallow"; you mean "invalid" as in syntactically invalid.  I think it's tricky to get both of those concepts clearly into one sentence; perhaps "List of valid IPv4 or IPv6 CIDRs to disallow"?

On the other hand, if it's clear that the 3rd column is only describing syntax, and not function, perhaps it was all OK as it was before.